### ISSUE TITLE
Blowing up in debug mode

### ISSUE THREAD

**Issue by kshedstrom** [NOAA-GFDL/SIS2#155]
Testing the Bering domain in debug mode with the ocean's hmin=-5, it is blowing up:
```
#1  0x1cab6bf in __sis_tracer_registry_MOD_set_massless_sis_tracers
        at //import/c1/AKWATERS/kate/ESMG/ESMG-configs/src/SIS2/src/SIS_tracer_registry.F90:439
#2  0x203c4c3 in __sis_transport_MOD_finish_ice_transport
        at //import/c1/AKWATERS/kate/ESMG/ESMG-configs/src/SIS2/src/SIS_transport.F90:290
#3  0xdc35ed in complete_ist_transport
        at //import/c1/AKWATERS/kate/ESMG/ESMG-configs/src/SIS2/src/SIS_dyn_trans.F90:755
#4  0xdc5eb2 in __sis_dyn_trans_MOD_sis_dynamics_trans
        at //import/c1/AKWATERS/kate/ESMG/ESMG-configs/src/SIS2/src/SIS_dyn_trans.F90:402
#5  0x183669b in __ice_model_mod_MOD_update_ice_dynamics_trans
        at //import/c1/AKWATERS/kate/ESMG/ESMG-configs/src/SIS2/src/ice_model.F90:328
#6  0x1837beb in __ice_model_mod_MOD_update_ice_model_slow
        at //import/c1/AKWATERS/kate/ESMG/ESMG-configs/src/SIS2/src/ice_model.F90:156
#7  0x8d47b1 in coupler_main
        at //import/c1/AKWATERS/kate/ESMG/ESMG-configs/src/coupler/coupler_main.F90:1037
#8  0x8e4d8d in main
        at //import/c1/AKWATERS/kate/ESMG/ESMG-configs/src/coupler/coupler_main.F90:314
```
This is with 72 cores, with or without a mask table. Using 4 cores in totalview, it runs past this time (2 hours), no problem.

**Comment by MJHarrison-GFDL** [comment]
I get the same error using gcc/8.3.0 . gcc/7.3.0 runs normally.

**Comment by Hallberg-NOAA** [comment]
SIS2 PR #156 corrects the use of an uninitialized array at this point in the code, and seems likely to address this issue.

**Comment by kshedstrom** [comment]
Yes, this does indeed get fixed by PR #156 
Thanks!
